test/system/450-interactive.bats

```diff
@@ -102,4 +102,29 @@ function teardown() {
     is "$output" "hello$CR" "-i=false: no warning"
 }
 
+
+@test "podman run -l passthrough-tty" {
+    skip_if_remote
+
+    if ! tty -s; then
+        skip "not running in a tty"
+    fi
+
+    # check if conmon supports -l passthrough for ttys, we can drop this check once conmon gets in the CI images.
+    run_podman info -f '{{.Host.Conmon.Path}}'
+    conmon_path="$output"
+
+    if ! $conmon_path -l passthrough --cid foo --cuuid foo --runtime /bin/true; then
```

Comment:
Do you know what version of conmon supports this, and where it is in the bodhi pipeline? I just spun up a rawhide VM, and even in updates-testing, I can't get a conmon that supports this. I would like to hold this PR until I can run this test against a truly working conmon.

Reply:
it is not released yet: https://github.com/containers/conmon/pull/465

Fine by me, let's hold it until the next conmon release